processed 3 tasks

task 2 'run'. lines 32-44:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: None,
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 6)],
}
